### ISSUE TITLE
Implement a more efficient way to handle closure operation requests that have more than one coding

### ISSUE THREAD

**Issue by JohnTimm** [IBM/FHIR#2162]
One step towards a more efficient way to handle more than one coding in a closure operation request is to add a closure method signature to the `FHIRTermServiceProvider` SPI that supports multiple codes. For example:
```
Map<Code, Set<Concept>> closure(CodeSystem codeSystem, Set<Code> codes);
```
and a closure method signature to `FHIRTermService`:
```
Map<Coding, Set<Concept>> closure(Set<Coding> codings);
```
See: https://chat.fhir.org/#narrow/stream/179202-terminology/topic/ibm.20as.20a.20fhir.20terminology.20server for more info

**Comment by lmsurpre** [comment]
I confirm that the new closure variants (`closure(Set<Coding> codings)` and `closure(CodeSystem codeSystem, Set<Code> codes)`) have been added to FHIRTermService.
`closure(Set<Coding> codings)` is covered by a unit test and is now used by the ClosureOperation.

`closure(CodeSystem codeSystem, Set<Code> codes)` doesn't seem to be covered by any tests, but its a one-line implementation the delegates to `closure(CodeSystem codeSystem, Set<Code> codes)` and I confirmed that both our registry-based and graph-based implementations are covered by FHIRTermServiceProviderTest.testClosure2().

One minor thing I noticed is that one variant of the closure method (`Set<Concept> closure(CodeSystem codeSystem, Code code)`) is missing javadoc, so that might be good to add.
